.github/workflows/ci.yaml

```diff
@@ -47,17 +47,14 @@ jobs:
               run: |
                   sudo echo "127.0.0.1" `hostname` | sudo tee -a /etc/hosts
 
+            - name: Configure debug logging
+              if: runner.debug == '1'
```

Comment:
I think it makes sense to describe what this flag is and how it can be enabled...

Not sure if some prose in ci.yaml makes sense or also pointing to this PR where one could see the GH screenshot etc